docs/advanced/checked-build.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Checked Builds
+redirect_from:
+  - /Checked_Build/
```

Comment:
You don't need that redirect, those top level URLs are a relic of the old MediaWiki based website.